static/app/views/performance/browser/webVitals/utils/useProjectWebVitalsQuery.tsx

```diff
@@ -24,10 +24,12 @@ export const useProjectWebVitalsQuery = ({transaction, tag}: Props = {}) => {
         'p75(measurements.cls)',
         'p75(measurements.ttfb)',
         'p75(measurements.fid)',
+        'p75(transaction.duration)',
         'count()',
         'failure_count()',
         'p95(transaction.duration)',
         'eps()',
+        'project',
```

Comment:
do we need to query for 'project' here? I think it's breaking some of the other pages because it changes the response to be grouped by project, particularly when the selected project has no results.

We can either update the other pages to handle this new edge case or update pageOverview to just use the project from the useProjects hook

Reply:
Yeah, I removed the project field from the query. We now only display the view transaction button when there is one project selected.